src/components/tree-item/tree-item.scss

```diff
@@ -177,7 +183,7 @@
 }
 
 // dropdown expanded and not selected
-:host([has-children][expanded]:not([selected])) > .node-container {
+:host([has-children][expanded]:not([selected]):not([selection-mode="none"])) > .node-container {
```

Comment:
nitpick: some of these selectors are getting pretty complex. Maybe at some point we move this logic to the tsx file?

Reply:
Yeah, I went with the simplest option for now. I think we'll have to do a pass on complex selectors and see which ones are better off as dynamic classes vs. their styling equivalent. cc @benelan